lms/djangoapps/django_comment_client/management/commands/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+'''
```

Comment:
Use double-quotes for docstrings, per the style guide [1].

[1] https://github.com/edx/edx-platform/wiki/Python-Guidelines